### ISSUE TITLE
Let Heroic handle game launch

### ISSUE THREAD

**Issue by redromnon** [Heroic-Games-Launcher/heroic-bash-launcher#3]
### Feature
A simpler and more straightforward method for launching games is to make the Heroic launch the game. This means that the generated launch scripts don't include the launch command anymore, will not actually launch games and look clean.

Heroic will have full control of launching the game and will also include pre-launch or post-launch operations like creating wine prefix if the game is run for the first time, DXVK/VKD3D setup, cloud-save syncing, etc.
**In short, running the game's launch script should perform the same function as Play Now button of a game.**

### Implementation
Under `#Command` in the launch script, necessary arguments like the appid will be passed to Heroic so that Heroic launches that game. For this, command-line arguments will have to be implemented in Heroic.

**Comment by Nocccer** [comment]
Makes this not heroic bash launcher obsolete ?

I thought the clue behind bash launcher is to launch games without starting heroic.

**Comment by redromnon** [comment]
You're right. This may truly make Bash Launcher obsolete.

Bash Launcher exists because users couldn't launch games without opening the Heroic app. This tool wouldn't have existed if Heroic included command-line arguments to launch games directly as Lutris does.

So the question is - should we continue the development of Bash Launcher (the way it's supposed to be i.e. launch command in the bash script) or introduce arguments to Heroic.

**Comment by Nocccer** [comment]
The goal is to great a script with launches legendary with the commands/args people setup in heroic.
Mainly this is for steam non-game compatiblity. So user can download/setup games in heroic and add them easy to here steam library.

**Comment by redromnon** [comment]
Alright, let's go ahead with this then.
